Erro no teste:  TramiteBlocoExternoLimiteTest

**Descreva o erro**

Verificar erro no teste TramiteBlocoExternoLimiteTest

Erro: 
..............................F................................  63 / 176 ( 35%)
............................................................... 126 / 176 ( 71%)
..................................................              176 / 176 (100%)
 
Time: 3.19 hours, Memory: 54.00 MB
 
There was 1 failure:
 
1) TramiteBlocoExternoLimiteTest::test_verificar_envio_tramite_em_bloco
ERROR: 255
make: *** [Makefile:195: test-functional] Error 255

-----

**Para reproduzir**
Passos para reproduzir o comportamento:
1. Vá para '...'
2. Clique em '....'
3. Role para baixo até '....'
4. Veja o erro

-----

**Comportamento esperado**
Uma descrição clara e concisa do que você esperava que acontecesse.

-----

**Screenshots**
Se aplicável, adicione capturas de tela para ajudar a explicar seu problema. Sugerimos o uso do Peek (linux) ou ScreenToGif (windows) para captura.

-----

**Contexto da falha (Por favor, complete a seguinte informação):**
 - Versão do sistema: [ex: 1.0.1]
 - Navegador: [ex. chrome, safari]
 - Perfil do Usuário: [ex: colaborador, externo, coordenador, administrador, etc.]
 - Banco de dados: [MySQL, SQLServer, Oracle ou PostgreSQL]
-----

**Informações adicionais**
Adicione qualquer outro contexto sobre o problema aqui.

-----
**Log de Erros**
Adicione o log de erros registrado pelo sistema. Verifique o log registrado no servidor, através da funcionalidade Infra> Log

--- a/tests_sei4/funcional/tests/TramiteBlocoExternoLimiteTest.php
+++ b/tests_sei4/funcional/tests/TramiteBlocoExternoLimiteTest.php
@@ -170,21 +170,30 @@ class TramiteBlocoExternoLimiteTest extends CenarioBaseTestCase
      */
     public function test_verificar_envio_tramite_em_bloco()
     {
-        $this->acessarSistema(
-            self::$remetente['URL'],
-            self::$remetente['SIGLA_UNIDADE'],
-            self::$remetente['LOGIN'],
-            self::$remetente['SENHA']
-        );
-        $this->paginaCadastrarProcessoEmBloco->navegarListagemBlocoDeTramite();
+      $this->acessarSistema(
+          self::$remetente['URL'],
+          self::$remetente['SIGLA_UNIDADE'],
+          self::$remetente['LOGIN'],
+          self::$remetente['SENHA']
+      );
+      $this->paginaCadastrarProcessoEmBloco->navegarListagemBlocoDeTramite();
+      $novoStatus = $this->paginaCadastrarProcessoEmBloco->retornarTextoColunaDaTabelaDeBlocos();
+
+      if (self::$tramitar == true) {
+        $this->waitUntil(function ($testCase) {
+          sleep(5);
+          $testCase->refresh();
+          $novoStatus = $this->paginaCadastrarProcessoEmBloco->retornarTextoColunaDaTabelaDeBlocos();
+          $this->assertNotEquals('Aguardando Processamento', $novoStatus);
+          return true;
+        }, PEN_WAIT_TIMEOUT);
+        
         $novoStatus = $this->paginaCadastrarProcessoEmBloco->retornarTextoColunaDaTabelaDeBlocos();
+        $this->assertEquals(utf8_encode("Concluído"), $novoStatus);
+      } else {
+        $this->assertEquals("Aberto", $novoStatus);
+      }
 
-        if (self::$tramitar == true) {
-            $this->assertEquals(utf8_encode("Concluído"), $novoStatus);
-        } else {
-            $this->assertEquals(utf8_encode("Aberto"), $novoStatus);
-        }  
-
-        $this->sairSistema();
+      $this->sairSistema();
     }
 }

--- a/tests_sei41/funcional/tests/TramiteBlocoExternoLimiteTest.php
+++ b/tests_sei41/funcional/tests/TramiteBlocoExternoLimiteTest.php
@@ -170,25 +170,30 @@ class TramiteBlocoExternoLimiteTest extends CenarioBaseTestCase
      */
     public function test_verificar_envio_tramite_em_bloco()
     {
+      $this->acessarSistema(
+          self::$remetente['URL'],
+          self::$remetente['SIGLA_UNIDADE'],
+          self::$remetente['LOGIN'],
+          self::$remetente['SENHA']
+      );
+      $this->paginaCadastrarProcessoEmBloco->navegarListagemBlocoDeTramite();
+      $novoStatus = $this->paginaCadastrarProcessoEmBloco->retornarTextoColunaDaTabelaDeBlocos();
+
+      if (self::$tramitar == true) {
+        $this->waitUntil(function ($testCase) {
+          sleep(5);
+          $testCase->refresh();
+          $novoStatus = $this->paginaCadastrarProcessoEmBloco->retornarTextoColunaDaTabelaDeBlocos();
+          $this->assertNotEquals('Aguardando Processamento', $novoStatus);
+          return true;
+        }, PEN_WAIT_TIMEOUT);
         
-        self::$remetente = $this->definirContextoTeste(CONTEXTO_ORGAO_A);
-        self::$destinatario = $this->definirContextoTeste(CONTEXTO_ORGAO_B);
-
-        $this->acessarSistema(
-            self::$remetente['URL'],
-            self::$remetente['SIGLA_UNIDADE'],
-            self::$remetente['LOGIN'],
-            self::$remetente['SENHA']
-        );
-        $this->paginaCadastrarProcessoEmBloco->navegarListagemBlocoDeTramite();
         $novoStatus = $this->paginaCadastrarProcessoEmBloco->retornarTextoColunaDaTabelaDeBlocos();
+        $this->assertEquals(utf8_encode("Concluído"), $novoStatus);
+      } else {
+        $this->assertEquals("Aberto", $novoStatus);
+      }
 
-        if (self::$tramitar == true) {
-            $this->assertEquals(utf8_encode("Concluído"), $novoStatus);
-        } else {
-            $this->assertEquals(utf8_encode("Aberto"), $novoStatus);
-        }  
-
-        $this->sairSistema();
+      $this->sairSistema();
     }
 }

--- a/tests_super/funcional/tests/TramiteBlocoExternoLimiteTest.php
+++ b/tests_super/funcional/tests/TramiteBlocoExternoLimiteTest.php
@@ -176,21 +176,30 @@ class TramiteBlocoExternoLimiteTest extends CenarioBaseTestCase
      */
     public function test_verificar_envio_tramite_em_bloco()
     {
-        $this->acessarSistema(
-            self::$remetente['URL'],
-            self::$remetente['SIGLA_UNIDADE'],
-            self::$remetente['LOGIN'],
-            self::$remetente['SENHA']
-        );
-        $this->paginaCadastrarProcessoEmBloco->navegarListagemBlocoDeTramite();
+      $this->acessarSistema(
+          self::$remetente['URL'],
+          self::$remetente['SIGLA_UNIDADE'],
+          self::$remetente['LOGIN'],
+          self::$remetente['SENHA']
+      );
+      $this->paginaCadastrarProcessoEmBloco->navegarListagemBlocoDeTramite();
+      $novoStatus = $this->paginaCadastrarProcessoEmBloco->retornarTextoColunaDaTabelaDeBlocos();
+
+      if (self::$tramitar == true) {
+        $this->waitUntil(function ($testCase) {
+          sleep(5);
+          $testCase->refresh();
+          $novoStatus = $this->paginaCadastrarProcessoEmBloco->retornarTextoColunaDaTabelaDeBlocos();
+          $this->assertNotEquals('Aguardando Processamento', $novoStatus);
+          return true;
+        }, PEN_WAIT_TIMEOUT);
+        
         $novoStatus = $this->paginaCadastrarProcessoEmBloco->retornarTextoColunaDaTabelaDeBlocos();
+        $this->assertEquals(utf8_encode("Concluído"), $novoStatus);
+      } else {
+        $this->assertEquals("Aberto", $novoStatus);
+      }
 
-        if (self::$tramitar == true) {
-            $this->assertEquals(utf8_encode("Concluído"), $novoStatus);
-        } else {
-            $this->assertEquals(utf8_encode("Aberto"), $novoStatus);
-        }  
-
-        $this->sairSistema();
+      $this->sairSistema();
     }
 }

